tests/e2e/docs/guest_cluster_setup.md

```diff
@@ -38,6 +38,14 @@
 
 ![WCP non shared DS 3](images/non-shared_ds_policy_step_2.png)
 
+### 4. Create Thick Provisioning Policy
```

Comment:
we need to mention that both the following are vsan based policies

Reply:
Updated the same